pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -2204,17 +2220,19 @@ func (a *ApplicationBackupController) backupResources(
 	backup.Status.BackupPath = GetObjectPath(backup)
 	backup.Status.Stage = stork_api.ApplicationBackupStageFinal
 	backup.Status.FinishTimestamp = metav1.Now()
-	if backup.Status.FailedVolCount > 0 {
+	if isPartialBackup {
 		backup.Status.Status = stork_api.ApplicationBackupStatusPartialSuccess
-	} else {
+	}
+	if backup.Status.FailedVolCount == 0 {
 		backup.Status.Status = stork_api.ApplicationBackupStatusSuccessful
 	}
 	if len(backup.Spec.NamespaceSelector) != 0 && len(backup.Spec.Namespaces) == 0 {
 		backup.Status.Reason = fmt.Sprintf("Namespace label selector [%s] did not find any namespaces with selected labels for backup", backup.Spec.NamespaceSelector)
 	} else {
-		if backup.Status.FailedVolCount > 0 {
+		if isPartialBackup {
```

Comment:
Any reason we need this as this checks are already taken care in #line2226

Reply:
The other check is happening in backupVolumes stage, this is in backupResources stage. Here also the status of the backupCR is being updated along with the reason. So, made changes accordingly with respect to updation of reason in case of partial backup